src/PagesBody/ProfilePage/ProfilePage.js

```diff
@@ -1,16 +1,19 @@
 import propTypes from 'prop-types';
+import { useContext } from 'react';
 
 import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
 import FakeButton from '../../Atoms/FakeButton/FakeButton';
 import ProfileCard from '../../Molecules/ProfileCard/ProfileCard';
+import { UserDataContext } from '../../Context/UserDataContext';
 
-const ProfilePage = ({ userId }) => {
+const ProfilePage = () => {
+  const userData = useContext(UserDataContext);
```

Comment:
What if the context data is not available? Is there any other way to check the user id? Thinking about authorization also - this component should check if the user that entered this page and is trying to access these resources, is the currently logged in user.

Reply:
Authentication checked. Context data availability is at the moment  more complicated subject.